app/views/candidate_interface/guidance/index.html.erb

```diff
@@ -0,0 +1,80 @@
+          </tr><tr class="govuk-table__row">
+            <td class="govuk-table__cell"><%= CycleTimetable.show_summer_recruitment_banner.to_s(:govuk_date) %></td>
+            <td class="govuk-table__cell">The time training providers have to make a decision about your application is reduced from 40 working days to 20 working days.</td>
+          </tr>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell"><%= CycleTimetable.apply_1_deadline.to_s(:govuk_date_and_time) %></td>
+            <td class="govuk-table__cell">If you’ve not already applied for teacher training, you will not be able to apply after <%= CycleTimetable.apply_1_deadline.to_s(:govuk_date) %>.<br><br> If you’ve already applied, but your applications were unsuccessful, you can continue to apply for courses until <%= CycleTimetable.apply_2_deadline.to_s(:govuk_date_and_time) %>.</td>
+          </tr>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell"><%= CycleTimetable.reject_by_default.to_s(:govuk_date_and_time) %></td>
+            <td class="govuk-table__cell">The last day for training providers to make a decision on all applications for courses starting in September <%= @recruitment_cycle_year %>.</td>
+          </tr>
+        </tbody>
+      </table>
+    <% else %>
+      <h1 class="govuk-heading-l">The application process for courses starting in September <%= @recruitment_cycle_year %></h1>
+      <p class="govuk-body">For courses starting in September <%= @recruitment_cycle_year %>, you can start applying a year before they start in October <%= @recruitment_cycle_year - 1 %>.</p>
+      <p class="govuk-body">Most teacher training courses are one year and start in September when the academic year in England starts. Part time courses are usually 2 years.</p>
+
+      <br>
+
+      <table class="govuk-table">
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Date and time</th>
+            <th scope="col" class="govuk-table__header">What happens</th>
+          </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell"><%= CycleTimetable.find_opens.to_s(:govuk_date_and_time) %></td>
+            <td class="govuk-table__cell">Start <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses" class="govuk-link">finding postgraduate teacher training courses</a> you might want to apply to.</td>
```

Comment:
Couple of tweaks:
* use the `govuk_link_to` helper
* lookup the URL from the locale file instead of hardcoding it (should be there already)